jsparagus/actions.py

```diff
@@ -281,6 +281,8 @@ class Reduce(Action):
     table execution to resume shifting or replaying terms."""
     __slots__ = ['unwind']
 
+    unwind: Unwind
```

Comment:
I don't see the consumer of this field

Reply:
It is already present in this file, what surprised me is the fact that the type checker did not complain about the missing field declaration.